db/migrate/20201117232135_change_approved_in_users.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeApprovedInUsers < ActiveRecord::Migration[6.0]
+  def change
+    change_column_default :bookings, :approved, from: nil, to: false
```

Comment:
Is this where you auto set a bookings approved status to false?

Reply:
Yeah, that's the default value of false 👍